cgo_flags.go

```diff
@@ -7,14 +7,10 @@ package rocksdb
 // #cgo CPPFLAGS: -Iinternal/utilities/merge_operators/string_append
 // #cgo windows CPPFLAGS: -DOS_WIN
 // #cgo !windows CPPFLAGS: -DROCKSDB_PLATFORM_POSIX -DROCKSDB_LIB_IO_POSIX
-// #cgo darwin CPPFLAGS: -DOS_MACOSX
-// #cgo linux CPPFLAGS: -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE
+// #cgo darwin CPPFLAGS: -DOS_MACOSX -DROCKSDB_BACKTRACE
+// #cgo linux CPPFLAGS: -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE -msse -msse4.2
```

Comment:
I've replaced these flags with `-march=native` which will enable all supported instruction sets on the host machine (and this is what build_detect_platform emits in the absence of USE_SSE).